packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -128,14 +136,14 @@ contract Initializer_Test is Bridge_Initializer {
         }
     }
 
-    /// @dev Returns the number of contracts that are `Initializable` in `src/L1`.
-    function _getNumL1Initializable() internal returns (uint256 numContracts_) {
+    /// @dev Returns the number of contracts that are `Initializable` in `src/L1` and `src/L2`.
+    function _getNumInitializable() internal returns (uint256 numContracts_) {
         string[] memory command = new string[](3);
         command[0] = Executables.bash;
         command[1] = "-c";
         command[2] = string.concat(
             Executables.find,
-            " src/L1 -type f -exec basename {} \\;",
+            " src/L1 src/L2 -type f -exec basename {} \\;",
```

Comment:
Should this just look in all of `src` instead of specifying each directory?

Reply:
I would be down to open this up in the future to all contracts in `src` in a follow up PR

Reply:
That works for me, will defer you on how to track, feel free to resolve this otherwise